docs/source/xhighfive.rst

```diff
@@ -0,0 +1,148 @@
+.. Copyright (c) 2016, Wolf Vollprecht, Johan Mabille and Sylvain Corlay
+
+   Distributed under the terms of the BSD 3-Clause License.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+xhighfive
```

Comment:
I think it would look better in the docs to have HDF5 interface as the title, and then mention that these functions are implemented in the `xhighfive` header.

Reply:
What about the name of the include?

Reply:
I like `xhighfive.hpp` because it's cool, but `xhdf5.hpp` might be more descriptive. Both are good with me.

Reply:
I'll keep the `xhighfive.hpp` then, as it hint at the need to have HighFive.